tmc-langs-r/src/main/java/fi/helsinki/cs/tmc/langs/r/RPlugin.java

```diff
@@ -0,0 +1,158 @@
+package fi.helsinki.cs.tmc.langs.r;
+
+
+import fi.helsinki.cs.tmc.langs.AbstractLanguagePlugin;
+import fi.helsinki.cs.tmc.langs.abstraction.ValidationResult;
+import fi.helsinki.cs.tmc.langs.domain.ExerciseBuilder;
+import fi.helsinki.cs.tmc.langs.domain.ExerciseDesc;
+import fi.helsinki.cs.tmc.langs.domain.RunResult;
+import fi.helsinki.cs.tmc.langs.domain.TestDesc;
+import fi.helsinki.cs.tmc.langs.io.StudentFilePolicy;
+import fi.helsinki.cs.tmc.langs.io.sandbox.StudentFileAwareSubmissionProcessor;
+import fi.helsinki.cs.tmc.langs.io.zip.StudentFileAwareUnzipper;
+import fi.helsinki.cs.tmc.langs.io.zip.StudentFileAwareZipper;
+
+import fi.helsinki.cs.tmc.langs.utils.ProcessRunner;
+
+
+import com.google.common.base.Optional;
+import com.google.common.collect.ImmutableList;
+
+
```

Comment:
Some extra lines here and in a few other places. Check imports for correct grouping.